src/commands.js

```diff
@@ -163,3 +163,18 @@ export const helpSpecific = msg => {
     });
   }
 };
+
+// ex:
+// !emoji <name> <url>
+export const emoji = async msg => {
+  const { content, channel, guild } = msg;
+  const regex = /!emoji\s(.*)\s(.*)/;
+
+  try {
+    const { 1: name, 2: url } = regex.exec(content);
+    const newEmoji = await guild.createEmoji(url, name);
+    channel.send(`\`${newEmoji.name}\` ${newEmoji} has been added!`);
```

Comment:
Note to self, double quotes would be better for us since we have to escape single quotes so much.

On another note, if you can it would be better if you made this into an embed message so that when it fails, it'll show a red border. As seen here: https://github.com/sirMerr/camperbot/pull/8 (also just noticed something broke there, so it won't work if you try it right now)

Reply:
Oh right lol, it's still a thing though and I agree with the choice 👍